src/mcp/mcp_client.py

```
@@ -0,0 +1,379 @@
+    def _get_auth_config_id(self, mcp: str) -> Optional[str]:
+        """Get auth config ID for a specific MCP"""
+        if mcp not in self.mcp_name2instance:
+            return None
+            
+        mcp_instance: ComposioMCP = self.mcp_name2instance[mcp]
+        if mcp_instance.mcp_id is not None:
+            try:
+                mcp_server = self.composio.mcp.retrieve(mcp_instance.mcp_id)
+                auth_config_ids: List[str] = mcp_server.auth_config_ids
+                return auth_config_ids[0] if auth_config_ids else None
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)